CONTRIBUTING.md

```diff
@@ -66,20 +66,25 @@ We love pull requests. Here's a quick guide:
 
 1. Fork the repo.
 
-2. Run the tests. We only take pull requests with passing tests, and it's great
+2. Ensure you have the development requirements:
+
+   * node & npm
```

Comment:
* we should specify the versions
* we should specifiy that these must be installed without sudo